### PR TITLE
Task7

### PR DESCRIPTION
**Pull request by Mateen-Ahmad5**
Create a new data view page and display some data in the table.
The data shown must be fetched from an external API. 
Display the fetched data (e.g., posts, comments, or any relevant data) in a visually appealing table on the interactive web page.

### CODE REVIEW

File: data-view.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+
+async function fetchData() {
+    try {
+        const response = await fetch('https://jsonplaceholder.typicode.com/todos/');
+        if (!response.ok) {
+            throw new Error('Network response was not ok');
```
Comment (aliahmarawan8) [inline data-view.js]: If fetch fails will it not trigger the catch by default?

File: data-view.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        }
+        const data = await response.json();
+        displayData(data);
```
Comment (aliahmarawan8) [inline data-view.js:19]: Best practice is to keep the Function above this function you are going to use.

**Review by aliahmarawan8 — CHANGES_REQUESTED** [review]
Please made the required changes.

File: data-view.js (hunk 1 above)
Comment (Mateen-Ahmad5) [reply data-view.js]: done

File: data-view.js (hunk 2 above)
Comment (Mateen-Ahmad5) [reply data-view.js:19]: resolve

**Review by aliahmarawan8 — APPROVED** [review]

**Pull request merged**